frontend/lib/src/hostComm/types.ts

```diff
@@ -216,6 +216,18 @@ export type AppConfig = {
    * Enables custom string messages to be sent to the host
    */
   enableCustomParentMessages?: boolean
+
+  /**
+   * Whether the script should be rerun after a websocket disconnect/reconnect.
+   * If false, the script will never be rerun. If true, the script will be
```

Comment:
nit: `the script will never be rerun` -> `will not rerun when reconnecting`.
`Never` sounds like it will never ever do it again 😱

Reply:
Ah, yeah 😅 fixed